dashboard/public/weblab/footer.js

```diff
@@ -29,11 +29,11 @@ var applyFooter = function() {
     document.body.appendChild(footerContents.childNodes[0]);
   }
   if (window.encrypted_channel_id) {
-    var link = document.getElementById("pagefooter_view_code");
-    link.onclick = function() {
-      window.location.href =
-        link.href + "/" + window.encrypted_channel_id + "/view";
-      return false;
-    };
+    var viewCodeLink = document.getElementById("pagefooter_view_code");
```

Comment:
This is still so much 🍝 but still better than before 😂 Thanks for working through this gnarly part of our system!

Reply:
😂🤦🏻‍♀️ so true. And thanks for the help!